twisted/plugins/graphite_blueflood_plugin.py

```diff
@@ -30,7 +31,13 @@ class Options(usage.Options):
         ['user',          'u', '', 'Rackspace authentication username. Leave empty if no auth is required'],
         ['key',           'k', '', 'Rackspace authentication password'],
         ['auth_url',      '',  AUTH_URL, 'Auth URL'],
-        ['limit',         '',  0, 'Blueflood json payload limit, bytes. 0 means no limit']
+        ['limit',         '',  0, 'Blueflood json payload limit, bytes. 0 means no limit'],
+        ['config',        'c', None,
+         'Path to a configuration file. The file must be in INI format, with '
+         '[bracketed] sections. All sections other than '
+         '[blueflood_carbon_forwarder] will be ignored. If this option is not '
+         'specified, no config file will be processed. Options specified in '
+         'the config file override values on the command line.']
```

Comment:
@shintasmith @ChandraAddala I updated the description of the `interval` option.
IMO, we shouldn't remove those command-line options. Override behavior is already documented in the `config` option, so that's not a problem. If we need to explain it more, or make it more prominent, I can add documentation to the README file.
The main problem with removing those options is because I made only minimal changes to the code to support loading from a config file. It actually just writes over the values given on the command line, using the same `options` object. See [here](https://github.com/rackerlabs/blueflood-carbon-forwarder/pull/16/files#diff-220c2e413d1e1469bd26294d376aa1b8R131). In order to load options from the config file but prevent them from being given on the command line would take a lot more work. I wouldn't be able to use the simple write-over solution anymore.